activation/nipost.go

```diff
@@ -349,7 +378,57 @@ func membersContainChallenge(members []types.Member, challenge types.Hash32) (ui
 	return 0, fmt.Errorf("challenge is not a member of the proof")
 }
 
-func (nb *NIPostBuilder) getBestProof(ctx context.Context, challenge types.Hash32) (types.PoetProofRef, *types.MerkleProof, error) {
+// TODO(mafa): remove after next poet round; https://github.com/spacemeshos/go-spacemesh/issues/4753
+func (nb *NIPostBuilder) addPoet111ForPubEpoch1(ctx context.Context) error {
+	// because poet 111 had a hardware issue when challenges for round 0 were submitted, no node could submit to it
+	// 111 was recovered with the poet 110 DB, so all successful submissions to 110 should be able to be fetched from there as well
+
+	client111, ok := nb.poetProvers["https://poet-111.spacemesh.network"]
+	if !ok {
+		// poet 111 is not in the list, no action necessary
+		return nil
+	}
+
+	nb.log.Info("pub epoch 1 mitigation: poet 111 is in the list of poets, adding it to the state as well")
+	client110 := nb.poetProvers["https://poet-110.spacemesh.network"]
```

Comment:
why add poet 110 as well?

Reply:
PoET 110 isn't added.

The logic is as follows: if the configuration contains both 110 and 111, but the state only contains 110 duplicate the state of 110 for 111.

Reply:
does it really matter? doesn't adding 111 only suffice?

Reply:
I can't follow. PoET 111 was restored with a copy of the state of PoET 110, someone who didn't submit to 110 won't be able to receive a proof from 111?